src/main/java/org/anystub/Base.java

```diff
@@ -598,7 +599,8 @@ public void save() throws IOException {
                 throw new IOException("stub file isn't created");
         }
 
-        try (FileWriter output = new FileWriter(file)) {
+        try (FileOutputStream out = new FileOutputStream(file);
+             OutputStreamWriter output = new OutputStreamWriter(out, StandardCharsets.UTF_8)) {
```

Comment:
is it not required in load?

Reply:
As far as I learned during load encoding is being autodetected by YAML library. I hadn't problem with load for cassettes written this way.